quantmsio/core/common.py

```
@@ -1,8 +1,15 @@
+"""
+Common constants and schemas for quantmsio.
+This module provides mapping dictionaries, column lists, and schemas used across the library.
+"""
+
 from quantmsio import __version__
 from quantmsio.core.format import PSM_FIELDS, FEATURE_FIELDS, IBAQ_FIELDS, PG_FIELDS
 import pyarrow as pa
+from typing import Dict, List, Set, Union
```

Ruff (0.8.2)

`typing.Union` imported but unused

Remove unused import: `typing.Union`

(F401)